### PR TITLE
Arranged timeout router variables into one place

### PR DESCRIPTION
**Pull request by bfallonf**
From: #8900 

@JacobTanenbaum as discussed. WDYT?

### CODE REVIEW

File: architecture/topics/router_environment_variables.adoc [hunk 1]
```diff
@@ -81,18 +69,46 @@ endif::[]
...
+|`ROUTER_CLIENT_FIN_TIMEOUT` | 1s | Controls the TCP FIN timeout period for the client connecting to the route. If the FIN sent to close the connection is not answered within the given time, HAProxy will close the connection anyway.  This is harmless if set to a low value and uses fewer resources on the router.
+|`ROUTER_DEFAULT_CLIENT_TIMEOUT`| 30s | Length of time within which a client has to acknowledge or send data.
+|`ROUTER_DEFAULT_CONNECT_TIMEOUT`| 5s | The maximum connect time.
+|`ROUTER_DEFAULT_SERVER_FIN_TIMEOUT` | 1s | Controls the TCP FIN timeout from the router to the pod backing the route.
+|`ROUTER_DEFAULT_SERVER_TIMEOUT`| 30s | Length of time within which a server has to acknowledge or send data.
+|`ROUTER_DEFAULT_TUNNEL_TIMEOUT` | 1h | Length of time till which TCP or WebSocket connections will remain open. If you have websockets/tcp
+connections (and any time HAProxy is reloaded), the old HAProxy processes
+will "linger" around for that period.
+|`ROUTER_SLOWLORIS_HTTP_KEEPALIVE`| 300s | Set the maximum time to wait for a new HTTP request to appear. If this is set too low, it can confuse browsers and applications not expecting a small `keepalive` value.
```
Comment (akostadinov) [inline architecture/topics/router_environment_variables.adoc]: Would be nice if here we add something like `see note below` or something to that effect.

**Comment by bfallonf** [comment]
Good plan, @akostadinov . I have updated with that.

@openshift/team-documentation PTAL

File: architecture/topics/router_environment_variables.adoc [hunk 2]
```diff
@@ -85,18 +73,47 @@ endif::[]
...
+
+`TimeUnits` are represented by a number followed by the unit: `us`
+*(microseconds), `ms` (milliseconds, default), `s` (seconds), `m` (minutes), `h`
+*(hours), `d` (days).
+
+The regular expression is: [1-9][0-9]*(us\|ms\|s\|m\|h\|d)
+
+[cols="2,2,6"]
+|===
+|`ROUTER_BACKEND_CHECK_INTERVAL` | 5000ms | Length of time between subsequent "liveness" checks on backends.
```
Comment (mburke5678) [inline architecture/topics/router_environment_variables.adoc]: s/"liveness"/_liveness_

File: architecture/topics/router_environment_variables.adoc [hunk 3]
```diff
@@ -85,18 +73,47 @@ endif::[]
...
+|`ROUTER_CLIENT_FIN_TIMEOUT` | 1s | Controls the TCP FIN timeout period for the client connecting to the route. If the FIN sent to close the connection is not answered within the given time, HAProxy will close the connection anyway.  This is harmless if set to a low value and uses fewer resources on the router.
```
Comment (mburke5678) [inline architecture/topics/router_environment_variables.adoc]: s/anyway/

File: architecture/topics/router_environment_variables.adoc [hunk 4]
```diff
@@ -85,18 +73,47 @@ endif::[]
...
+|`ROUTER_DEFAULT_CLIENT_TIMEOUT`| 30s | Length of time within which a client has to acknowledge or send data.
+|`ROUTER_DEFAULT_CONNECT_TIMEOUT`| 5s | The maximum connect time.
+|`ROUTER_DEFAULT_SERVER_FIN_TIMEOUT` | 1s | Controls the TCP FIN timeout from the router to the pod backing the route.
+|`ROUTER_DEFAULT_SERVER_TIMEOUT`| 30s | Length of time within which a server has to acknowledge or send data.
+|`ROUTER_DEFAULT_TUNNEL_TIMEOUT` | 1h | Length of time till which TCP or WebSocket connections will remain open. If you have websockets/tcp
```
Comment (mburke5678) [inline architecture/topics/router_environment_variables.adoc]: s/till/

Comment (mburke5678) [reply architecture/topics/router_environment_variables.adoc]: s/within which/that ??

File: architecture/topics/router_environment_variables.adoc [hunk 5]
```diff
@@ -85,18 +73,47 @@ endif::[]
...
+*(microseconds), `ms` (milliseconds, default), `s` (seconds), `m` (minutes), `h`
+*(hours), `d` (days).
+
+The regular expression is: [1-9][0-9]*(us\|ms\|s\|m\|h\|d)
+
+[cols="2,2,6"]
+|===
+|`ROUTER_BACKEND_CHECK_INTERVAL` | 5000ms | Length of time between subsequent "liveness" checks on backends.
+|`ROUTER_CLIENT_FIN_TIMEOUT` | 1s | Controls the TCP FIN timeout period for the client connecting to the route. If the FIN sent to close the connection is not answered within the given time, HAProxy will close the connection anyway.  This is harmless if set to a low value and uses fewer resources on the router.
+|`ROUTER_DEFAULT_CLIENT_TIMEOUT`| 30s | Length of time within which a client has to acknowledge or send data.
```
Comment (mburke5678) [inline architecture/topics/router_environment_variables.adoc]: s/within which/that ??

File: architecture/topics/router_environment_variables.adoc [hunk 6]
```diff
@@ -85,18 +73,47 @@ endif::[]
...
+|`ROUTER_DEFAULT_CONNECT_TIMEOUT`| 5s | The maximum connect time.
+|`ROUTER_DEFAULT_SERVER_FIN_TIMEOUT` | 1s | Controls the TCP FIN timeout from the router to the pod backing the route.
+|`ROUTER_DEFAULT_SERVER_TIMEOUT`| 30s | Length of time within which a server has to acknowledge or send data.
```
Comment (mburke5678) [inline architecture/topics/router_environment_variables.adoc]: s/within which/that ??

File: architecture/topics/router_environment_variables.adoc [hunk 7]
```diff
@@ -85,18 +73,47 @@ endif::[]
...
+|`ROUTER_DEFAULT_TUNNEL_TIMEOUT` | 1h | Length of time till which TCP or WebSocket connections will remain open. If you have websockets/tcp
+connections (and any time HAProxy is reloaded), the old HAProxy processes
+will "linger" around for that period.
```
Comment (mburke5678) [inline architecture/topics/router_environment_variables.adoc]: s/"linger"/_linger_

File: architecture/topics/router_environment_variables.adoc [hunk 8]
```diff
@@ -85,18 +73,47 @@ endif::[]
...
+|`ROUTER_SLOWLORIS_HTTP_KEEPALIVE`| 300s | Set the maximum time to wait for a new HTTP request to appear. If this is set too low, it can confuse browsers and applications not expecting a small `keepalive` value. Additive. See note box below for more information.
```
Comment (mburke5678) [inline architecture/topics/router_environment_variables.adoc]: s/confuse/cause problems with ??

File: architecture/topics/router_environment_variables.adoc [hunk 9]
```diff
@@ -85,18 +73,47 @@ endif::[]
...
+|`ROUTER_SLOWLORIS_TIMEOUT` | 10s | Length of time the transmission of an HTTP request can take.
+|`RELOAD_INTERVAL` | 5s | The minimum frequency the router is allowed to reload to accept new changes.
+|`ROUTER_METRICS_HAPROXY_TIMEOUT` | 5s |
```
Comment (mburke5678) [inline architecture/topics/router_environment_variables.adoc]: Description is missing

**Comment by mburke5678** [comment]
@bfallonf A few nits. Otherwise LGTM

**Comment by bfallonf** [comment]
Thanks @mburke5678 . I simply moved the content, and didn't think of checking the wording. I've changed to your suggestions. I'll merge.

**Pull request merged**